ArtOfIllusion/src/artofillusion/RotateViewTool.java

```diff
@@ -208,13 +211,13 @@ private void dragRotateSpace(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown && !e.isShiftDown())
+		if (mouseButtonTwo(e) && controlDown)
 		{
 			view.tilting = true;
 			tilt(e, view, clickPoint);
 			return;
 		}
-		else if (controlDown && e.isShiftDown()){
+		else if (mouseButtonThree(e) && e.isShiftDown()){
```

Comment:
Here too.
```suggestion
 		else if (mouseButtonThree(e) && e.isShiftDown() && e.isControlDown()){
```
Should probably check if the selection by mouseButtonThree has alredy been done....